tests/e2e/bootstrap_test.go

```diff
@@ -115,6 +138,46 @@ var _ = Describe("E2E - Bootstrapping node", Label("bootstrap"), func() {
 			Expect(err).To(Not(HaveOccurred()))
 		})
 
+		By("Adding SeedImage", func() {
```

Comment:
I think that it should be conditioned with `if isoBoot == "true" {`, like the part you removed in this PR.